src/content/en/fundamentals/performance/optimizing-content-efficiency/webfont-optimization.md

```diff
@@ -271,16 +272,266 @@ Lazy loading of fonts carries an important hidden implication that may delay tex
 The "race" between the first paint of page content, which can be done shortly 
 after the render tree is built, and the request for the font resource is what 
 creates the "blank text problem" where the browser might render page layout but 
-omits any text. The actual behavior differs between various browsers:
-
-* Safari delays text rendering until the font download is complete.
-* Chrome and Firefox delay font rendering for up to 3 seconds, after which they use a fallback font. After the font download has finished, they re-render the text with the downloaded font.
-* IE immediately renders with the fallback font if the request font is not yet available, and re-renders the text after the font download is complete.
+omits any text.
+
+The next section describes a number of options for customizing this default behavior.
+
+### Preload your Webfont resources
+
+If there's a high probability that your page will need a specific Webfont hosted
+at a URL you know in advance, you can take advantage of a new web platform
+feature: [`<link rel="preload">`](https://developers.google.com/web/fundamentals/performance/resource-prioritization).
```

Comment:
The linter will warn that these references include the fully qualified developers.google.com path. You might want to go back and drop this back to `/web/fundamentals/..`.

Reply:
Yup, will do.